src/configure/configure.ts

```diff
@@ -173,8 +182,12 @@ class Orchestrator {
                 this.inputs.azureSession = getAzureSession();
             }
             const repoAnalysisResult = await this.getRepositoryAnalysis();
-            await this.getSelectedPipeline(repoAnalysisResult);
-
+            await this.setPipelineType();
+            // Right click on not resource not supported for Azure pipeline
+            if (resourceNode && this.pipelineType === PipelineType.AzurePipeline) {
```

Comment:
it seems `resourceNode` is always going to be null as `analyzeNode()` always return null

Reply:
line 268
```return this.inputs.targetResource.resource;```
It returns the resource in case of right click scenario.

Reply:
Ah I missed that addition

Reply:
As mentioned in another comment, instead of using `resourceNode` variable, we should rely on a function called `isResourceAlreadySelected` which will internally check if  this.context['isResourceAlreadySelected'] is true

Reply:
yes, that we can do
Or anyway as we are initializing the targetResource as a part of inputs, that can be used as well.
Because for cases, where we require resource details when it is selected, we will have to use the ``` inputs.targetResource.resource ```